src/FffGcodeWriter.cpp

```diff
@@ -2498,7 +2497,7 @@ bool FffGcodeWriter::processSupportInfill(const SliceDataStorage& storage, Layer
             const coord_t support_line_width = default_support_line_width * (combine_idx + 1);
 
             Polygons support_polygons;
-            VariableWidthPaths wall_toolpaths;
+            VariableWidthPaths dummy_wall_toolpaths;
```

Comment:
I would assign the pre-generated wall_toolpaths to this variable. Since the `generate()` still makes some use of them.
so in effect `VariableWidthPaths wall_toolpaths = part.wall_toolpaths`

Reply:
Yeah I wanted to ask about that ... why a copy and not a reference?

Reply:
I think it I did that because this function was const, and the generate could modify the wall tool paths.
Might be worth looking into if we can use reference

Reply:
What function is const? part.(infill_)wall_toolpaths isn't? combineSupportInfillLayers isn't?

Reply:
No but the FFGodewriter processSingleLayerInfill and processMultiLayerInfill is.

Reply:
quick call?

Reply:
Turns out I can't do it either :-p